### PR TITLE
SCAN Negative Counts

### PR DESCRIPTION
**Pull request by joverlee521**
Formative requested that we include negative counts by day as well so they can calculate the percent positives.

### CODE REVIEW

**Comment by joverlee521** [comment]
[Add negative counts to `shipping.scan_hcov19_positives_v1`](https://trello.com/c/KM7SlilR/706-add-negative-counts-to-shippingscanhcov19positivesv1)

**Comment by joverlee521** [comment]
> The view is called "hcov19_positives" but we now show negatives, too. Perhaps we ought to rename it so it's more accurate?

I wondered about the name too. Would `scan_hcov19_result_counts` be more appropriate?

**Comment by kairstenfay** [comment]
> > The view is called "hcov19_positives" but we now show negatives, too. Perhaps we ought to rename it so it's more accurate?
> 
> I wondered about the name too. Would `scan_hcov19_result_counts` be more appropriate?

Yes, I think that's an ideal name.

**Comment by joverlee521** [comment]
I've renamed the view and also took the opportunity to remove the unused `seattle_neighborhood_districts` view.

File: schema/deploy/shipping/views.sql [hunk 1]
```diff
@@ -2250,56 +2251,38 @@ create or replace view shipping.scan_hcov19_positives_v1 as
...
+        count(*) filter (where hcov19_result = 'negative' and county = 'yakima') as yakima_county_negatives,
+        count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county is null) as other_positives,
+        count(*) filter (where hcov19_result = 'negative' and county is null) as other_negatives
     from scan_hcov19_results
     group by hcov19_result_release_date
 ;
 
-comment on view shipping.scan_hcov19_positives_v1 is
+comment on view shipping.scan_hcov19_result_counts_v1 is
   'A view of counts of hcov19 positives from the SCAN project grouped by date results were released.';
```
Comment (jccraft-uw) [inline schema/deploy/shipping/views.sql:2268]: Nipik: Update the comment from positives to results.

Comment (joverlee521) [reply schema/deploy/shipping/views.sql:2268]: Oh, will update! Thank you 👍

File: schema/sqitch.plan [hunk 2]
```diff
@@ -206,3 +206,5 @@ shipping/views [shipping/views@2020-07-17] 2020-07-20T20:56:28Z Jover Lee <jover
 
 shipping/views [shipping/views@2020-07-20] 2020-07-29T18:37:58Z Kairsten Fay <kfay@fredhutch.org> # Add kit received to SCAN enrollments
 @2020-07-29 2020-07-29T18:42:32Z Kairsten Fay <kfay@fredhutch.org> # Schema as of 29 July 2020
+
+shipping/views [shipping/views@2020-07-29] 2020-08-05T20:40:42Z Jover Lee <joverlee@fredhutch.org> # Add negative counts to shipping.scan_hcov19_positives_v1
```
Comment (kairstenfay) [inline schema/sqitch.plan:210]: I believe this comment is out of date

File: schema/deploy/shipping/views.sql [hunk 3]
```diff
@@ -2250,33 +2251,38 @@ create or replace view shipping.scan_hcov19_positives_v1 as
...
 
 -- Even if it's just aggregate counts of hcov19 positives,
```
Comment (kairstenfay) [inline schema/deploy/shipping/views.sql:2270]: "positives" should be "results" here

But honestly a pretty minor change so it's probably fine to leave

File: lib/seattleflu/id3c/api/routes.py [hunk 4]
```diff
@@ -109,11 +109,11 @@ def get_scan_demographics(session):
 @authenticated_datastore_session_required
 def get_scan_positives(session):
```
Comment (kairstenfay) [inline lib/seattleflu/id3c/api/routes.py:110]: Should this function be renamed?

Comment (joverlee521) [reply lib/seattleflu/id3c/api/routes.py:110]: Gernot had asked to add it to the existing endpoint, so I assume they don't want to change the API call on their end.

**Pull request merged**